authentication/authentication.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
```

Comment:
In spirit of hype-driven development, how about we put this file under `v1`, so it would be `/authentication/v1/authentication.proto`

Reply:
there's a better solution that I intentionally do not comply because I don't think there will be any version change for this app once it's reached completion (completion as in no one wants to move on and develop it)

the rule of versioning resides on the package name, hence it will be `/authentication/authentication.v1.proto` with package name of `authentication.v1` instead of having v1 on the directory level. see: https://docs.buf.build/lint/rules#package_version_suffix